build/verify-test-urls.sh

```diff
@@ -0,0 +1,43 @@
+#! /bin/bash
+
+# Automatically exit on error
+set -e
+
+EXIT_CODE=0
+
+# Keep short because GitHub doesn't wrap it
+ANNOTATION="Test URLs missing.
```

Comment:
Maybe its just me, the wording sounds flipped
```suggestion
ANNOTATION="Missing Test URLs.
```

Reply:
No, it’s like VIRUS DETECTED, right? 

“Missing X” would be followed by a list of X, like:

```
Missing keys:

- A
- Q
``` 

but here it works like:

```
# A.tsx

Keys missing (from this file)
```

Reply:
Ok